routes/users.js

```diff
@@ -1,19 +1,28 @@
 var express = require('express');
 const crypto = require('crypto')
-var router = express.Router();
+const jwtValidate = require('express-jwt')
 
+const secret = require('../config').secret
 const User = require('../models/user')
-const AccessToken = require('../models/access-token')
 const Address = require('../models/address')
-const TokenValidator = require('../middlewares/token')
+const generateToken = require('../utils/token-generator')
 
+var router = express.Router();
 
-function generateToken() {
-  const date = new Date
-  return crypto.createHash('md5').update(date.getTime().toString()).digest('hex')
-}
+router.use(jwtValidate({secret, algorithms: ['HS256']}))
+router.use((err, req, res, next) => {
```

Comment:
don't write this in route file. use it in a middleware. and use that middleware function

Reply:
you need to verify if that user exist in our db or not